src/superannotate/lib/core/service_types.py

```diff
@@ -0,0 +1,74 @@
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Union
+
+from lib.core.exceptions import AppException
+from pydantic import BaseModel
+from pydantic import Extra
+
+
+class UserLimits(BaseModel):
+    super_user_limit: int
+    project_limit: int
+    folder_limit: int
+
+    def has_enough_slots(self, count: int):
+        if count > self.super_user_limit:
```

Comment:
The order of print seems to be incorrect, should be folder, project and super-admin

Reply:
This logic will be moved to the usecases